indico/modules/rb/models/rooms.py

```diff
@@ -95,9 +95,10 @@ class Room(versioned_cache(_cache, 'id'), db.Model, Serializer):
         db.Integer,
         db.ForeignKey('roombooking.photos.id')
     )
-    name = db.Column(
+    #: Verbose name for the room (long)
+    verbose_name = db.Column(
         db.String,
-        nullable=False
+        nullable=True
```

Comment:
null vs empty string? I think in most places we use empty strings and `nullable=False`

Reply:
I personally think that when doing a DB query that involves checking whether there is a `verbose_name` or not, it's much cleaner to do a `.isnot(None)`. I know we don't normally do that for strings, but in this case we have to do checks on whether the field is set.

Reply:
In that case I would add a check constraint to prevent empty strings there. Otherwise I can guarantee that at some point some code will set it to an empty string when editing/updating ;)

Reply:
Done!